### ISSUE TITLE
Generating MSI fails

### ISSUE THREAD

**Issue by tfabraham** [BTDF/DeploymentFramework#288]
Deployment framework : 5.1
VS 2012
BTS 2013

GenericBizTalkMSI.wxs
  RedistComponentGroup.wxs
  "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\light.exe" "obj\Debug\GenericBizTalkMSI.wixobj" "obj\Debug\RedistComponentGroup.wixobj" -out "C:\Projects\US\GATEWAY\Gateway.Deployment\bin\Debug\Gateway-1.0.0.msi" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixUIExtension.dll" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixNetFxExtension.dll" -spdb
  Windows Installer Xml Linker version 3.7.1224.0
  Copyright (C) Outercurve Foundation. All rights reserved.

E:\delivery\Dev\wix37_public\src\ext\UIExtension\wixlib\LicenseAgreementDlg.wxs(32): error LGHT0103: The system cannot find the file 'License.rtf'. [C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj]
C:\Program Files (x86)\MSBuild\DeploymentFrameworkForBizTalk\5.0\BizTalkDeploymentFramework.WiXSetup.targets(251,5): error MSB3073: The command ""C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\light.exe" "obj\Debug\GenericBizTalkMSI.wixobj" "obj\Debug\RedistComponentGroup.wixobj" -out "C:\Projects\US\GATEWAY\Gateway.Deployment\bin\Debug\Gateway-1.0.0.msi" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixUIExtension.dll" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixNetFxExtension.dll" -spdb" exited with code 103. [C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj]
Done Building Project "C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj" (Installer target(s)) -- FAILED.

Build FAILED.

"C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj" (Installer target) (1) ->
(Installer target) -> 
  E:\delivery\Dev\wix37_public\src\ext\UIExtension\wixlib\LicenseAgreementDlg.wxs(32): error LGHT0103: The system cannot find the file 'License.rtf'. [C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj]
  C:\Program Files (x86)\MSBuild\DeploymentFrameworkForBizTalk\5.0\BizTalkDeploymentFramework.WiXSetup.targets(251,5): error MSB3073: The command ""C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\light.exe" "obj\Debug\GenericBizTalkMSI.wixobj" "obj\Debug\RedistComponentGroup.wixobj" -out "C:\Projects\US\GATEWAY\Gateway.Deployment\bin\Debug\Gateway-1.0.0.msi" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixUIExtension.dll" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixNetFxExtension.dll" -spdb" exited with code 103. [C:\Projects\US\GATEWAY\Gateway.Deployment\Gateway.Deployment.btdfproj]

#### This work item was migrated from CodePlex
CodePlex work item ID: '10217'
Vote count: '1'

**Comment by tfabraham** [comment]
[tfabraham@6/26/2013]
The error indicates that License.rtf cannot be found.  That file must be in your Deployment project folder next to the .btdfproj.  It is included in every new project created with the Add New Project wizard.

**Comment by tfabraham** [comment]
[UnknownUser@7/10/2013]

**Comment by tfabraham** [comment]
[fdesim@1/14/2014]
Tom,

I'm getting the exact same error when trying to build an MSI on the TFS build server.  I have License.rtf in my Deployment project folder and have also placed it in the project build folder in TFS.

E:\delivery\Dev\wix37_public\src\ext\UIExtension\wixlib\LicenseAgreementDlg.wxs (32): The system cannot find the file 'License.rtf'.

 C:\Program Files (x86)\MSBuild\DeploymentFrameworkForBizTalk\5.1\BizTalkDeploymentFramework.WiXSetup.targets (251): The command ""C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\light.exe" "obj\Debug\GenericBizTalkMSI.wixobj" "obj\Debug\RedistComponentGroup.wixobj" -out "C:\Builds\1\BizTalk\BTDFDemo1Test1\src\Test1.Deployment\bin\Debug\\Test1-.msi" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixUIExtension.dll" -ext "C:\Program Files (x86)\Deployment Framework for BizTalk\5.1\Developer\WiX\WixNetFxExtension.dll" -spdb" exited with code 103.

**Comment by tfabraham** [comment]
[tfabraham@1/14/2014]
Please check your build's working folder on the build server to see if the License.rtf is being pulled from TFS and is in the same folder as the .btdfproj.

**Comment by tfabraham** [comment]
[fdesim@1/14/2014]
I added License.rtf to the project and verified that it is in the same folder in TFS as .btdfproj.  This gave me a new error that is referenced in this post:  https://biztalkdeployment.codeplex.com/discussions/218533.  I was able to get things working by setting SuppressMSIValidation=True in C:\Program Files (x86)\MSBuild\DeploymentFrameworkForBizTalk\5.1\BizTalkDeploymentFramework.WiXSetup.targets, as suggested in that post.
